03_launch_mgmt_cluster.sh

```diff
@@ -412,15 +413,28 @@ apply_bm_hosts()
     local namespace="$1"
     pushd "${BMOPATH}"
 
-    list_nodes | make_bm_hosts
-    if [[ -n "$(list_nodes)" ]]; then
-        echo "bmhosts_crs.yaml is applying"
-        while ! kubectl apply -f "${WORKING_DIR}/bmhosts_crs.yaml" -n "${namespace}" &>/dev/null; do
-            sleep 3
-        done
-        echo "bmhosts_crs.yaml is successfully applied"
-    fi
+    local RETRY=10
+    while [[ "${RETRY}" -gt 0 ]]; do
+      echo "bmhosts_crs.yaml is applying"
+      list_nodes | make_bm_hosts
+      # check if we have a not empty manifests file
+      if [[ -s "${WORKING_DIR}/bmhosts_crs.yaml" ]]; then
```

Comment:
We're repeating `${WORKING_DIR}/bmhosts_crs.yaml` three times in the function, put it in a variable.